src/exportJarSteps/ExportJarTaskProvider.ts

```diff
@@ -6,22 +6,22 @@ import {
     TaskProvider, TaskRevealKind, TaskScope, TerminalDimensions,
 } from "vscode";
 import { createJarFile } from "../exportJarFileCommand";
-import { getExportJarTargetPath } from "../settings";
+import { Settings } from "../settings";
 import { IStepMetadata } from "./IStepMetadata";
 
 export class ExportJarTaskProvider implements TaskProvider {
 
     public static exportJarType: string = "exportjar";
 
     public static getTask(stepMetadata: IStepMetadata): Task {
-        const targetPathSetting: string = getExportJarTargetPath();
+        const targetPathSetting: string = Settings.getExportJarTargetPath();
         const defaultDefinition: IExportJarTaskDefinition = {
             type: ExportJarTaskProvider.exportJarType,
             targetPath: targetPathSetting,
             elements: [],
             mainMethod: undefined,
         };
-        const task: Task = new Task(defaultDefinition, TaskScope.Workspace, "DEFAULT_EXPORT", ExportJarTaskProvider.exportJarType,
+        const task: Task = new Task(defaultDefinition, TaskScope.Workspace, "exportJarTask", ExportJarTaskProvider.exportJarType,
```

Comment:
Again, this name is to display to the users, you should not give users a variable-like name